### ISSUE TITLE
Local function execution fails when the handler is in a subdirectory

### ISSUE THREAD

**Issue by marcelreppi** [kyma-project/cli#1122]
**Description**

When I place the source code of my Kyma function into a subdirectory like `src`, run `kyma run function` and go to my browser and go to `http://localhost:8080/` it crashes with the error `Error: Cannot find module '/kubeless/handler.js'`.
Based on the [documentation](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-06-function-configuration-file/) I would assume this should work.
Deploying this setup actually works fine in our Kyma cluster but locally the execution fails.

**Expected result**

The browser returns "Hello Serverless"

**Actual result**

There is an internal server error and the process crashes with the error `Error: Cannot find module '/kubeless/handler.js'`

**Steps to reproduce**

1. `kyma init function -d my-function`
2. `cd my-function`
3. `mkdir src`
4. `mv handler.js src`
5. Add `sourceHandlerName: src/handler.js` to `source` in the `config.yaml`
6. `kyma run function`
7. Open `http://localhost:8080/` in your browser

**Troubleshooting**

I tried different combinations of the `sourcePath` and `sourceHandlerName` fields but nothing worked.

**Comment by kwiatekus** [comment]
thank you @Cortey 
This fixes the problem.

I can reference `sourcePath` as `src` subfolder and run it locally
```yaml
name: my-function
namespace: default
runtime: nodejs14
source:
    sourceType: inline
    sourcePath: src
```
or I can reference Handler from `src` folder
```yaml
name: my-function
namespace: default
runtime: nodejs14
source:
    sourceType: inline
    sourceHandlerName: src/handler.js
```

**Comment by kwiatekus** [comment]
Fixed in Kyma CLI  -> version: `main-fd29944`
